src/pages/yujinLee/Main/Main.js

```diff
@@ -1,6 +1,49 @@
-import React from 'react';
+import Nav from '../../../components/Nav/Nav';
+import Feed from '../MyComponent/Feed';
+import FriendProfile from '../MyComponent/FriendProfile';
+import AsideFooter from '../MyComponent/AsideFooter';
+// import 'font-awesome/css/font-awesome.min.css';
+import '../Main/Main.scss';
+
 const Main = () => {
-  return <h1>Main입니다.</h1>;
+  return (
+    <div className="mainWrapper">
+      <Nav />
```

Comment:
`mainWrapper`로 네스팅하는 부분에서의 스타일링이 적용될 수 있기 때문에 Fragment로 분리!
```suggestion
    <>
      <Nav />
      <div className="mainWrapper">
```